src/libavrdude.h

```diff
@@ -60,6 +60,35 @@ typedef uint32_t pinmask_t;
 #define LIBAVRDUDE_SOFTFAIL (-3) // returned, eg, by avr_signature() if caller
                                  // might proceed with chip erase
 
+/* Message system */
+/* These 2 functions are supposed to be supplied by the application */
+
+int avrdude_message(int msglvl, const char *format, ...);
```

Comment:
Now that `avrdude_message()` is gone, perhaps remove it here and change comment above to This function is ...